testResources/reactTsProject/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "reacttsproject",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.3.1",
+    "react-dom": "^16.3.1",
+    "react-scripts-ts": "2.14.0"
+  },
+  "scripts": {
+    "start": "react-scripts-ts start",
+    "build": "react-scripts-ts build",
+    "test": "react-scripts-ts test --env=jsdom",
+    "eject": "react-scripts-ts eject"
+  },
+  "devDependencies": {
+    "@types/jest": "^22.2.3",
+    "@types/node": "^9.6.4",
+    "@types/react": "^16.3.9",
+    "@types/react-dom": "^16.0.5",
+    "typescript": "^2.8.1"
```

Comment:
These dependencies aren't actually used right? Maybe delete them so they are not accidentally installed.

Reply:
We don't use them, and they aren't installed either. I'd rather keep them here, because tests can also double as documentation - people might want to take a look there to see how things are _supposed to work_.